### ISSUE TITLE
on-the-fly extend strategies

### ISSUE THREAD

**Issue by malb** [fplll/fplll#431]
We should not bomb out when a user tries to use a dimension larger than what's in the strategy file but rather compute a strategy on the fly for large dimensions.

**Comment by joerowell** [comment]
Does this not assume access to the strategizer in fplll? I don't think fplll ships with those bits (i.e set_mdc.py is probably necessary, although I guess we could fix 64 cycles per node?)

**Comment by malb** [comment]
Nope, we'd guess the right block size/cost by extrapolation and just call the pruner
